src/components/matchHistory/MatchHistory.tsx

```diff
@@ -16,6 +19,10 @@ export const MatchHistory = React.memo(function MatchHistory() {
         return <Loading text="" />;
     }
 
+    // ADD TEST CODE HERE AS NEEDED

```

Comment:
remove this for now, we can put it back in when you are testing, but would rather not check this in